docs/_posts/2016-09-28-our-first-50000-stars.md

```diff
@@ -18,7 +18,7 @@ The T-shirts are super soft using American Apparel's tri-blend fabric; we also h
 * [Toddler T-Shirts](https://teespring.com/react-50000-stars-toddler)
 * [Baby Onesies](https://teespring.com/react-50000-stars-baby)
 
-Proceeds from the shirts will be donated to [CODE2040](http://www.code2040.org/), a nonprofit that creates access, awareness, and opportunities in technology for underrepresented minorities with a specific focus on Black and Latinx talent.
+Proceeds from the shirts will be donated to [CODE2040](http://www.code2040.org/), a nonprofit that creates access, awareness, and opportunities in technology for underrepresented minorities with a specific focus on Black and Latino talent.
```

Comment:
"Latinx" actually the correct term, see [the code2040 site](http://www.code2040.org/). It's a general-nuetral alternative to "Latino".